source/dubregistry/registry.d

```diff
@@ -250,22 +250,23 @@ class DubRegistry {
 
 	string getReadme(Json version_info, DbRepository repository)
 	{
-		string ret;
-		auto file = version_info["readmeFile"].opt!string;
-		try {
-			if (!file.startsWith('/')) return null;
-			auto rep = getRepository(repository);
-			logDebug("reading readme file for %s: %s", version_info["name"].get!string, file);
-			rep.readFile(version_info["commitID"].get!string, Path(file), (scope data) {
-				ret = data.readAllUTF8();
-			});
-			logDebug("reading readme file done");
-		} catch (Exception e) {
-			logDiagnostic("Failed to read README file (%s) for %s %s: %s",
-				file, version_info["name"].get!string,
-				version_info["version"].get!string, e.msg);
+		auto readme = version_info["readme"].opt!string;
+
+		// compat migration, read file from repo if README hasn't yet been stored in the db
```

Comment:
shouldn't we store it in the DB too then?

Reply:
To much work to change all the places, it just takes an hour or so to populate all packages in the regular update loop.